changelogs/fragments/9128-homebrew_cask-name-regex-fix.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - homebrew_cask - allow ``+`` symbol in homebrew cask name validation regex (https://github.com/ansible-collections/community.general/pull/9128).
```

The product name is written with a capital H:
```suggestion
  - homebrew_cask - allow ``+`` symbol in Homebrew cask name validation regex (https://github.com/ansible-collections/community.general/pull/9128).
```
(I also noticed that the module's docs themselves use both forms, they should be aligned at some point...)